Lib/email/mime/application.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, _data, _subtype='octet-stream',
                  _encoder=encoders.encode_base64, *, policy=None, **_params):
         """Create an application/* type MIME document.
 
-        _data is a string containing the raw application data.
+        _data contains the bytes for the raw application data.
```

Comment:
Operationally, it doesn't matter for `MIMEApplication`:

```
>>> from email.mime.application import MIMEApplication
>>> MIMEApplication('data')
<email.mime.application.MIMEApplication object at 0x1035e0e90>
>>> a = MIMEApplication('data')
>>> b = MIMEApplication(b'data')
>>> a.get_payload()
'ZGF0YQ==\n'
>>> b.get_payload()
'ZGF0YQ==\n'
```

although it still probably makes sense to promote bytes over strings here.

Reply:
I'd be happy to make the requested changes if i knew how. If there's an obvious way to edit the labels on the PR I haven't found it. (I have a vague recollection of asking about this same thing last time I submitted a PR for a Python repo, and never getting an answer). As for whether I've "checked the actual stdlib documentation," I'm not really sure what that means. I mean, I assume it's obvious that I filed the ticket in the first place because I was reading the [documentation](https://docs.python.org/3/library/email.mime.html) for the Python standard library and found it confusing and misleading (and I provided links to the sections to which I was referring). So presumably that's not what you're asking.

Maybe I should just confine my contributions to filing the occasional bug report and leave the PRs to the insiders who actually know what they're doing. 🤔

Reply:
> Maybe I should just confine my contributions to filing the occasional bug report and leave the PRs to the insiders who actually know what they're doing

You're doing great, and I thank you the effort to make this better for everyone!  I'm happy to help you get this one over the line.

I've gone ahead ahead and added the appropriate labels: `skip-news` as mentioned, but also backport labels to the [active maintenance (bug fix) releases](https://devguide.python.org/versions/), Python 3.10 and 3.11.

Reply:
> As for whether I've "checked the actual stdlib documentation," I'm not really sure what that means.

Your PR touches the docstrings for three standard library files, which is great, but it doesn't actually update the documentation that gets published at the link you included.  Can you also update [this file](https://raw.githubusercontent.com/python/cpython/main/Doc/library/email.mime.rst)?

Reply:
Ah! So in spite of [this language](https://docs.python.org/3/glossary.html#term-docstring):

> **docstring**
> A string literal which appears as the first expression in a class, function or module. While ignored when the suite is executed, it is recognized by the compiler and put into the __doc__ attribute of the enclosing class, function or module. Since it is available via introspection, **_it is the canonical place for documentation of the object_**. [emphasis added]
> 

... the same language is being maintained in two different places? That's unfortunate. I'll add the fix to the second place.

As for the labels, I still haven't ever been given an answer to my original question: How does a contributor without commit access add a label to a PR? I've looked everywhere for the interface I would need and it's very elusive.